.github/workflows/actions.yml

```diff
@@ -14,7 +14,9 @@ jobs:
       with:
         node-version: '14.x'
     - name: Install npm dependencies
-      run: npm install eslint
+      run: npm install eslint 
+    - name: Install eslint standard
+      run: npm install --save-dev eslint-config-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node
```

Comment:
`npm install --save-dev` will save the packages in `package.json`. Better run it once, locally, to populate `package.json` and then just call `npm install` , which will install the dependencies in `package.json`.